docs/openhabian-backup.md

```diff
@@ -1,141 +1,148 @@
-    Use your personal user instead (that you have created at the beginning of your openHABian installation or `openhabian` by default).
-    Installation tasks including post-package-installation changes (edits) of the Amanda config files, require to use the `root` user.
-    Any ordinary user (such as your personal one) can execute commands on behalf of root (and with root permission) by prepending `sudo` to the command.
-    As yourself, prepend `sudo -u backup` to execute the following commands as the "backup" user.
-
-# Installation
-
-These notes were written for an interactive installation run.
-Note openHABian provides the new ["auto backup" feature](https://github.com/openhab/openhabian/blob/main/docs/openhabian.md#auto-backup).
+---
+layout: documentation
+title: openHABian Backup
+source: https://github.com/openhab/openhabian/blob/main/docs/openhabian-backup.md
+---
+
+# Backing up openHABian
+
+Accidents happen, electronics wear out, SD cards die, but you don't need to wait until then to prepare for it.
+In fact you should not ever do that!
+
+Reconfiguring openHAB and openHABian from scratch can be incredibly painful (trust me, I speak from experience).
+If you take a couple of basic precautions and implement some basic backup features when you setup openHABian, you can greatly reduce the chance that you would need to restart from scratch.
+
+We will provide some basic guidance below, please take the time to implement it, we promise you will regret it if you don't!
+
+## What to Prepare For
+
+In an ideal world, you should have a spare for *every* component that you use in you smart home system.
+This means having a backup SD card, RPi, router, switch, and any external addons (e.g. ZWave stick).
+
+Since you probably won't actually do that (but you should) here are some simple things that can reduce the headache if it is simply a part of your openHABian instance fails.
+
+### SD Card Failure
+
+One of the most common points of failure in a Raspberry Pi is an SD card failing.
+This is why we *strongly* recommend using SD cards that are labelled "Endurance" as they are made for applications like this.
+
+#### SD Mirroring
+
+We have a bultin option to mirror SD cards so that you can always have a backup ready.
+
+If you plan to mirror you SD card using the auto-backup features in openHAB, you should purchase 2 of the same model of SD card when you setup your system.
+A different model of SD card may work, but sometimes models differ slightly in size causing issues.
+If you can't find the same model, play it safe and buy a model larger than what you currently have to use for your backup.
+
+::: caution Moving the Root Filesystem
+Moving your system root to a USB stick or SSD is unsupported and dangerous.
+Besides USB sticks in particular suffer from the same issues as SD cards.
+SSD's do as well but to a lesser extent.
+In summary, just don't do it!
```

Comment:
I know you don't want to write this but a major reason for the harshish old text is that there's still a number of oh users that boot from SSDs and keep insisting on telling other that to be a good idea.
While not wrong technically in itself, we know that they are neither implementing this generically nor are they supporting that so it's always us to pick up the slack.
We really must give more background info and make the reasoning clearer here, else too many inexperienced readers will again fall victim to those telling them SSDs are *the* solution.